packages/neutrino-preset-web/index.js

```diff
@@ -85,8 +84,10 @@ module.exports = (neutrino) => {
     .when(process.env.NODE_ENV !== 'test', () => neutrino.use(chunk))
     .target('web')
     .context(neutrino.options.root)
+	 .entry('vendor')
+      .add(require.resolve('./polyfills.js'))
```

Comment:
Originally this refactoring had a separate entry point `.entry('polyfills')` which produced one more file chunk. In this case polyfills could be disabled easy by name. But I decided to make changes more soft and joined vendor with polyfills togehter. Now splitting vendor and polyfills seems not a bad idea from the configuration perspective.